### ISSUE TITLE
"This flow is no longer active and it should have been purged from memory" errors

### ISSUE THREAD

**Issue by StewAlexander-com** [ntop/ntopng#2743]
@emanuele-f 
Errors showing up under the `Detected Alerts` menu:
```
HTTP/1.1 500 Internal server error Content-Type: text/html Connection: close   Script "/usr/share/ntopng/scripts/lua/pro/enterprise/flow_alerts_explorer.lua" returned an error:
/usr/share/ntopng/scripts/lua/modules/alert_utils.lua:1595: bad argument #1 to 'for iterator' (table expected, got boolean)
```
System info:
![image](https://user-images.githubusercontent.com/48565067/61877160-6dc19a80-aebc-11e9-85d3-f194b840bd92.png)

**Comment by StewAlexander-com** [comment]
Appears related to ticket ##2725

**Comment by StewAlexander-com** [comment]
These messages are coming in very quickly, at a rate of 2 or 3 a second, repeated several times a minute...

![image](https://user-images.githubusercontent.com/48565067/61877695-8b433400-aebd-11e9-98d9-b40720679d98.png)

**Comment by simonemainardi** [comment]
please, update to the latest version and check if this is happening again.

**Comment by StewAlexander-com** [comment]
After deleting a few arp messages crossing vlan boundaries these showed up again...

![image](https://user-images.githubusercontent.com/48565067/61892839-a7a19980-aeda-11e9-95d2-56ff552e2727.png)
----------
![image](https://user-images.githubusercontent.com/48565067/61892876-c43dd180-aeda-11e9-9b2b-231abc4a7583.png)

**Comment by simonemainardi** [comment]
@StewAlexanderACC can you please tell if these errors are gone in the lates build? afe2e51 should have fixed this

**Comment by simonemainardi** [comment]
Fixed as part of afe2e51. Before the fix, flows weren't purged for dynamic interfaces.
